packages/moonstone/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The following is a curated list of changes in the Enact moonstone module, newest
 
 ### Fixed
 
+- `moonstone/SlotItem` to change `vertical-align` from `top` to `middle` to align the center
```

Comment:
This change log entry is pretty implementation-specific. We don't normally call out CSS changes even to root components. Is it necessary to share this level of detail with developers?

Reply:
Hmm.. I think so. I'll remove it.